garth/users/settings.py

```
@@ -95,4 +102,9 @@ def get(cls, /, client: http.Client | None = None):
             "/userprofile-service/userprofile/user-settings"
         )
         assert isinstance(settings, dict)
-        return cls(**camel_to_snake_dict(settings))
+        data = camel_to_snake_dict(settings)
+        if 'user_sleep_windows' in data:
+            data['user_sleep_windows'] = [
+                UserSleepWindow(**window) for window in data['user_sleep_windows']
```

Ruff (0.8.2)

Line too long (82 > 79)

(E501)